src/providers/editVideoSheet/hooks.tsx

```diff
@@ -83,7 +83,7 @@ export const useEditVideoSheetTabData = (tab?: EditVideoSheetTab) => {
 
   return {
     tabData: normalizedData,
-    loading: tab.isDraft ? false : loading,
+    loading: tab.isDraft ? false : loading || !(thumbnailPhotoUrl && mediaUrl),
```

Comment:
I'm still getting weird results when I'm opening videos. 
For example: 
![copy](https://user-images.githubusercontent.com/51168865/123108631-e039ad00-d43a-11eb-9263-6a7c26bb5bae.gif)

I think this suggestion should resolve the problem
```suggestion
    loading: tab.isDraft ? false : loading || (!thumbnailPhotoUrl && !mediaUrl)
```
Could you confirm that? It works for me at least

Reply:
This is super weird, for me it works well in both versions + those conditions are equal. But ok, I'll fix it anyway.

Reply:
I don't think they are equal
for example
```js
!(false && true) // this will return true
(!false && !true) // this will return false
```

Reply:
Yes, sorry, you are right!